scipy/stats/_stats_py.py

```diff
@@ -7231,13 +7239,23 @@ def chisquare(f_obs, f_exp=None, ddof=0, axis=0):
     the asymptotic distribution is not chi-square, in which case this test
     is not appropriate.
 
+    For Pearson's chi-squared test, the total observed and expected counts must match
+    for the p-value to accurately reflect the probability of observing such an extreme
+    value of the statistic under the null hypothesis.
```

Comment:
```suggestion
    Pearson's chi-squared test assumes the total number of observations is fixed (thus
    giving the default ``k-1`` degrees of freedom), and so an error will be generated if
    the total observed and expected counts are not equal; set ``sum_check=False`` to
    disable this check.
```

Reply:
Could you summarize the intent of the change? The intent of the original is just to set the stage for the rest of the paragraph, not document `sum_check` or `ddof`.

Reply:
Well, the intent was to say why they 'must match' (the reason being the degrees of freedom), and to more explicitly give the reasoning behind the sum_check. However, what you had there was good, so if it is more opaque to you, then ignore my suggestion.

Reply:
The suggestion is good content; it's just different information from what I had in in mind for the beginning of this paragraph. Thanks!